src/graphql/root/mutation/ln-invoice-create-on-behalf-of-recipient.ts

```diff
@@ -21,15 +21,15 @@ const LnInvoiceCreateOnBehalfOfRecipientMutation = GT.Field({
     input: { type: GT.NonNull(LnInvoiceCreateOnBehalfOfRecipientInput) },
   },
   resolve: async (_, args) => {
-    const { recipient, amount, memo } = args.input
-    for (const input of [recipient, amount, memo]) {
+    const { recipientWalletId, amount, memo } = args.input
+    for (const input of [recipientWalletId, amount, memo]) {
       if (input instanceof Error) {
         return { errors: [{ message: input.message }] }
       }
     }
 
     const result = await addInvoiceForRecipient({
-      recipient,
+      recipientWalletPublicId: recipientWalletId,
```

Comment:
Unrelated to this change (which looks good), it looks like this mutation can have a `satoshis` property in the return under the `LnInvoice` type but we don't add one here so the mutation always returns a `null` value for that property

```gql
mutation lnInvoiceCreateOnBehalfOfRecipient($input: LnInvoiceCreateOnBehalfOfRecipientInput!) {
  lnInvoiceCreateOnBehalfOfRecipient(input: $input) {
    errors {
      message
    }
    invoice {
      paymentRequest
      paymentHash
      paymentSecret
      satoshis
    }
  }
}
```

Note: you may need to change the `recipientWalletId` value below based on local setup

```json
{"input": {"recipientWalletId": "60b8b880-f7ba-41e9-a4de-ea7cca19912b", "amount": "1000", "memo": "This is a test"}}
```

Reply:
I don't see the mutation exposing that field. Should it?

Reply:
I don't think we need it, we can keep it consistent with the `lnNoAmountInvoiceCreateOnBehalfOfRecipient` mutation which returns a `LnNoAmountInvoice` with no `satoshis` property.

But I guess my question is more should we be able to enter `satoshis` as a property in the GraphQL interface for `lnInvoiceCreateOnBehalfOfRecipient` given that it's a valid property on the `LnInvoice` type returned from that mutation's definition even though we haven't implemented `satoshis` in the mutation implementation?

For context, the difference between the 2 right now is that the [`LnInvoice` type](https://github.com/GaloyMoney/galoy/blob/749544702c63e6e19ddaf0f89aec0be4faabd1c0/src/graphql/main/schema.graphql#L587) in the `LnInvoicePayload` has a `satoshis` property and the [`LnNoAmountInvoice` type](https://github.com/GaloyMoney/galoy/blob/749544702c63e6e19ddaf0f89aec0be4faabd1c0/src/graphql/main/schema.graphql#L611) in the `LnNoAmountInvoicePayload` doesn't.